models/servicenow__ticket_enhanced.sql

```diff
@@ -119,8 +122,8 @@ select
   task.close_notes,
   task.comments,
   task.comments_and_work_notes,
-  task.company_link,
-  task.company_value, 
+  task.company_link, 
+  task.company_value,  
```

Comment:
One last note, I realized there is an [open height ticket](https://fivetran.height.app/T-445955) with a request from multiple customers where they require the `*_link` fields to have the display name provided. In this case, if you look at `company_link` and `company_value` you find that there is no actual `company_name` provided. This makes it extremely difficult to extract insights from the data when all you have are long formed sys_ids.

My request would be to bring in all the display names for the `*_link` fields in this table. You already brought in a few, like you did for assignee [here](https://github.com/fivetran/dbt_servicenow/blob/3df2be51d87514f5753d24c833b3e2b4b72a2051/models/servicenow__ticket_enhanced.sql#L92-L98). This means you will likely need to include a few more staging models to join in the enriched information. If you do not know what source tables you need to leverage, you can find the origin table for the display_name within the `*_link` field. See the screenshot below how you can find the company_link base table is `core_company` and the assigned_to_link base table is `sys_user` as you have already included.

![image](https://github.com/fivetran/dbt_servicenow/assets/74217849/3d621575-8cc5-40b4-82a5-2d5e53f477ed)

Reply:
Please note that some of the `*_link` fields will not require bringing in a new table. For example, the `parent_link` is just the `task` table and is fine to just leverage the `parent_value` for the result, and the `opened_by_link` just references `sys_user` which you already have as a staging model and just needs a new join to bring in the appropriate user information.

Reply:
Added new staging models cmdb_ci, cmdb_ci_service, sys_user_group. Added *_name to the respective *_link fields in the task_enhanced model. For all the user-based ones, display names are already provided in addition to the following fields from the user table. (example problem_fix_by_value >> problem_fixer_name, problem_fixer_email, problem_fixer_manager_value, problem_fixer_department_value, problem_fixer_roles)